src/resources/resources.controller.ts

```diff
@@ -43,6 +81,23 @@ export class ResourcesController {
     @ApiOperation({
         summary: "Gets all resources added by a team given a teamId (int)",
     })
+    @ApiResponse({
+        status: HttpStatus.OK,
+        description: "Returns all the resources for a particular team",
+        isArray: true,
+        type: TeamResourceAddedByResponse,
+    })
+    @ApiResponse({
+        status: HttpStatus.NOT_FOUND,
+        description: "Invalid teamId",
+        type: NotFoundErrorResponse,
+    })
+    @ApiParam({
+        name: "teamId",
+        required: true,
+        description: "Voyage team ID",
+        example: 1,
+    })
```

Comment:
Can you add response for undocumented 401 (unauthorized)
![Screenshot 2024-01-12 at 3 44 37 PM](https://github.com/chingu-x/chingu-dashboard-be/assets/80556079/66727ced-ec59-4bc9-be5b-c89a289e0782)
 here?

Reply:
Fixed -- I could have sworn I tried adding them in before but couldn't get them to trigger, but now they work fine. Maybe that time I just forgot to reload Swagger....